web/packages/teleterm/src/services/appUpdater/autoUpdatesStatus.ts

```diff
@@ -206,69 +227,56 @@ function findMostCompatibleToolsVersion(
-        kind: 'managing-cluster';
-        /** URI of the managing cluster. */
-        clusterUri: RootClusterUri;
-        /** Clusters considered during version resolution. */
-        clusters: Cluster[];
-        /** Clusters from which version information could not be retrieved. */
-        unreachableClusters: UnreachableCluster[];
-      }
-    | {
-        /** Updates determined by the most compatible clusters available. */
-        kind: 'most-compatible';
-        /** Clusters considered during version resolution. */
-        clusters: Cluster[];
-        /**
-         * Clusters from which version information could not be retrieved.
-         * If non-empty, the update is not automatically downloaded.
-         * */
-        unreachableClusters: UnreachableCluster[];
-        /**
-         * Indicates whether a cluster configured to manage updates is unreachable
-         * or not managing updates (toolsAutoUpdate: false).
-         */
-        skippedManagingClusterUri: string;
-      };
-};
+  /**
+   * Source of the update:
+   * - `env-var` - TELEPORT_TOOLS_VERSION configures app version.
+   * - `managing-cluster` - updates are managed by a manually configured cluster.
+   * - `highest-compatible` - updates are determined by the highest compatible version available.
+   */
+  source: 'env-var' | 'managing-cluster' | 'highest-compatible';
+  /**
+   * Represents the options considered during the auto-update version resolution process.
+   * If updates are configured via the environment variable, all fields will be empty or undefined.
+   */
+  options: AutoUpdatesOptions;
+}
 
-export type AutoUpdatesDisabled =
-  | {
-      enabled: false;
-      /** `TELEPORT_TOOLS_VERSION` is 'off'. */
-      reason: 'disabled-by-env-var';
-    }
-  | {
-      enabled: false;
-      /** There is no cluster that could manage updates. */
-      reason: 'no-cluster-with-auto-update';
-      /** Clusters considered during version resolution. */
-      clusters: Cluster[];
-      /** Clusters from which version information could not be retrieved. */
-      unreachableClusters: UnreachableCluster[];
-    }
-  | {
-      enabled: false;
-      /**
-       * There are clusters that could manage updates, but they specify
-       * incompatible client tools versions.
-       */
-      reason: 'no-compatible-version';
-      /** Clusters considered during version resolution. */
-      clusters: Cluster[];
-      /** Clusters from which version information could not be retrieved. */
-      unreachableClusters: UnreachableCluster[];
-    };
+export interface AutoUpdatesDisabled {
+  enabled: false;
+  /**
+   * Reason the updates are disabled:
+   * `disabled-by-env-var` - `TELEPORT_TOOLS_VERSION` is 'off'.
+   * `no-cluster-with-auto-update` - there is no cluster that could manage updates.
+   * `managing-cluster-unable-to-manage` - the manually selected managing cluster is either
+   * unreachable or stopped autoupdates.
```

Comment:
"Stopped autoupdates" feels like too much of a simplification that might be unclear to someone who hasn't spent much time with the topic of auto-updates.

```suggestion
   * unreachable or it has since disabled autoupdates.
```